public/app/core/components/NestedFolderPicker/NestedFolderList.tsx

```diff
@@ -172,7 +175,9 @@ const getStyles = (theme: GrafanaTheme2) => {
       display: 'flex',
       position: 'relative',
       alignItems: 'center',
-      borderTop: `solid 1px ${theme.components.input.borderColor}`,
+      [':not(:first-child)']: {
+        borderTop: `solid 1px ${theme.components.input.borderColor}`,
```

Comment:
I feel like the contrast for this color is a bit high. It probably works better on a greyer background.